main.go

```diff
@@ -19,6 +19,7 @@ package main
 import (
 	"context"
 	"flag"
+	"github.com/openshift/ocm-agent-operator/controllers/fleetnotification"
```

Comment:
Very minor style nit - move to the lower import block.

Reply:
thanks for that... not sure why the IDE put them into such order